tools/cli/internal/openapi/oasdiff.go

```diff
@@ -417,6 +483,27 @@ func (o OasDiff) areSchemaIdentical(name string) bool {
 	return !ok
 }
 
+// arePathsIdenticalWithExcludeExtensions checks if the paths are identical with the extensions excluded
+func (o OasDiff) arePathsIdenticalWithExcludeExtensions(name string) (bool, error) {
```

Comment:
the code seems to be checking if the path are identical with and without extensions 🤔  is this correct? if yes , we should rename the function to reflect this logic